sofl/details_dialog.py

```
@@ -320,22 +337,16 @@ def set_cover(self, _source: Any, result: Gio.Task, *_args: Any) -> None:
         except GLib.Error:
             log_message("Error getting file path", logging.ERROR)
```

Ruff (0.12.2)

Undefined name `log_message`

(F821)